src/components/Proposals/ProposalPage/OPProposalPage/StandardProposalPage.tsx

```diff
@@ -11,12 +11,16 @@ export default async function StandardProposalPage({
   return (
     <div className="flex flex-col">
       <ProposalStateAdmin proposal={proposal} />
-      <div className="flex gap-8 lg:gap-16 justify-between items-start max-w-[76rem] flex-col md:flex-row md:items-start md:justify-between">
-        <ProposalDescription proposal={proposal} />
-        <div>
+      <div className="flex gap-0 md:gap-16 justify-between items-start max-w-[76rem] flex-col md:flex-row md:items-start md:justify-between">
+        <div className="w-full proposal-description pb-6 md:pb-0">
```

Comment:
For `proposal-description` you can use pb-[90px] or something like pb-22, since we already use tailwind this would be more inline with it.